modules/lib/src/main/resources/assets/js/app/inputtype/ui/selector/image/ImageEditor.ts

```diff
@@ -788,6 +751,11 @@ export class ImageEditor
         this.uploadButton.setEnabled(false).addClass('button-upload');
         standbyContainer.appendChildren(this.resetButton, this.uploadButton);
 
+        let rightContainer = new DivEl('right-container');
```

Comment:
1. Don't hesitate using const, thus we declare that variable is not going to change
2. Always set type of a variable, not only in new code, but also when you update some piece of code where vars are missing their types or have `let` instead of `const`

Reply:
My bad here. I extracted the code from the top toolbar to create the bottom one, so I wasn't paying attention to those details. Thanks!